src/overmapbuffer.cpp

```diff
@@ -71,7 +71,7 @@ int camp_reference::get_distance_from_bounds() const
 
 cata_path overmapbuffer::terrain_filename( const point_abs_om &p )
 {
-    return PATH_INFO::world_base_save_path_path() / string_format( "o.%d.%d", p.x(), p.y() );
+    return PATH_INFO::current_dimension_save_path_path() / string_format( "o.%d.%d", p.x(), p.y() );
 }
 
 cata_path overmapbuffer::player_filename( const point_abs_om &p )
```

Comment:
also add dimension stuff here

Reply:
Aha, thanks, I missed that the "player" part was all "seen" data, need to wrap up the last of those.